File: CS/HowToAccessApplicationModelInCode.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.ExpressApp;
using DevExpress.Persistent.BaseImpl;

namespace HowToAccessApplicationModelInCode.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            if (ObjectSpace.FindObject<Contact>(CriteriaOperator.Parse(
              "FirstName == 'Mary' && LastName == 'Tellitson'")) == null) {
                Contact contact = ObjectSpace.CreateObject<Contact>();
                contact.FirstName = "Mary";
                contact.LastName = "Tellitson";
                contact.Email = "tellitson@example.com";
                contact.Save();
            }
            if (ObjectSpace.FindObject<Contact>(CriteriaOperator.Parse(
             "FirstName == 'Andrew' && LastName == 'Fuller'")) == null) {
                Contact contact = ObjectSpace.CreateObject<Contact>();
                contact.FirstName = "Andrew";
                contact.LastName = "Fuller";
                contact.Email = "fuller@example.com";
                contact.Save();
            }
        }
    }
}

File: VB/HowToAccessApplicationModelInCode.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace HowToAccessApplicationModelInCode.Module
   Public Class Updater
	   Inherits ModuleUpdater
        Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
	  Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
		 MyBase.UpdateDatabaseAfterUpdateSchema()
		 If ObjectSpace.FindObject(Of Contact)(CriteriaOperator.Parse("FirstName == 'Mary' && LastName == 'Tellitson'")) Is Nothing Then
			 Dim contact As Contact = ObjectSpace.CreateObject(Of Contact)()
			 contact.FirstName = "Mary"
			 contact.LastName = "Tellitson"
			 contact.Email = "tellitson@example.com"
			 contact.Save()
		 End If
		 If ObjectSpace.FindObject(Of Contact)(CriteriaOperator.Parse("FirstName == 'Andrew' && LastName == 'Fuller'")) Is Nothing Then
			 Dim contact As Contact = ObjectSpace.CreateObject(Of Contact)()
			 contact.FirstName = "Andrew"
			 contact.LastName = "Fuller"
			 contact.Email = "fuller@example.com"
			 contact.Save()
		 End If
	  End Sub
   End Class
End Namespace
